### PR TITLE
Add binary file option to slather action

### PR DESCRIPTION
**Pull request by mAu888**
This adds the binary file flag to the slather command. This probably should only be merged once the new version was pushed, currently you'd have to build the gem yourself to benefit :)

### CODE REVIEW

File: lib/fastlane/actions/slather.rb [hunk 1]
```diff
@@ -32,6 +32,7 @@ def self.run(params)
         command += " --show" if params[:show]
         command += " --source-directory #{params[:source_directory].shellescape}" if params[:source_directory]
         command += " --output-directory #{params[:output_directory].shellescape}" if params[:output_directory]
+        command += " --binary-file #{params[:binary_file].shellescape}" if params[:binary_file]
```
Comment (KrauseFx) [inline lib/fastlane/actions/slather.rb]: Please add a newline after this line.

**Comment by KrauseFx** [comment]
Hey @mAu888, thanks for this pull request. Could you please make sure `rubocop` succeeds when running `rubocop -a`? Also, please squash the commits. Let me know once the PR is ready to be merged :+1:

**Comment by mAu888** [comment]
Done. As @neonichu pushed slather `2.0.0` (:tada:) this PR is good to go :)

**Comment by neonichu** [comment]
Would be great if he `--binary-basename` option could also make it in here :)

**Comment by mAu888** [comment]
Oh, didn't see that option yet. Sure, I'll add it as well.

**Comment by neonichu** [comment]
It was just added yesterday :grin:

**Comment by mAu888** [comment]
So I did a small reorganisation how the arguments are handled. If some more ruby fluent one could have a small look I would be very thankful, because me doing Ruby is like ...

No idea

**Comment by ohwutup** [comment]
:rocket: thanks!!!

**Pull request merged**